web/src/js/urlState.js

```diff
@@ -78,6 +78,6 @@ function updateUrlFromStore(store) {
 }
 
 export default function initialize(store) {
-    updateStoreFromUrl(store)
     store.subscribe(() => updateUrlFromStore(store))
+    updateStoreFromUrl(store)
```

Comment:
This problem should solve itself if we export both functions and test them independently. :)